dotcom-rendering/src/components/MostViewedRightWithAd.tsx

```diff
@@ -20,14 +20,17 @@ type Props = {
 const MAX_HEIGHT_PX = 1600;
 
 export const MostViewedRightWithAd = ({
-	display,
+	format,
 	isPaidContent,
 	renderAds,
 	shouldHideReaderRevenue,
 }: Props) => {
 	const componentDataAttribute = 'most-viewed-right-container';
 	const { renderingTarget } = useConfig();
 	const isApps = renderingTarget === 'Apps';
+	const isMedia =
+		format.design === ArticleDesign.Video ||
+		format.design === ArticleDesign.Audio;
```

Comment:
Same as above with `isDark` / `hasDarkColourScheme` and `'light' | 'dark'` prop.